docs/INSTALL.md

```diff
@@ -2,38 +2,63 @@
 
 Dendrite can be run in one of two configurations:
 
- * A cluster of individual components, dealing with different aspects of the
-   Matrix protocol (see [WIRING.md](./WIRING.md)). Components communicate with
-   one another via [Apache Kafka](https://kafka.apache.org).
+* **Polylith mode**: A cluster of individual components, dealing with different
+  aspects of the Matrix protocol (see [WIRING.md](WIRING.md)). Components communicate with each other using internal HTTP APIs and [Apache Kafka](https://kafka.apache.org). This will almost certainly be the preferred model
+  for large-scale deployments.
 
- * A monolith server, in which all components run in the same process. In this
-   configuration, Kafka can be replaced with an in-process implementation
-   called [naffka](https://github.com/matrix-org/naffka).
+* **Monolith mode**: All components run in the same process. In this mode,
+   Kafka is completely optional and can instead be replaced with an in-process
+   lightweight implementation called [Naffka](https://github.com/matrix-org/naffka). This will usually be the preferred model for low-volume, low-user
+   or experimental deployments.
+
+Regardless of whether you are running in polylith or monolith mode, each Dendrite component that requires storage has its own database. Both Postgres
+and SQLite are supported and can be mixed-and-matched across components as
+needed in the configuration file.
+
+Be advised that Dendrite is still developmental and it's not recommended for
+use in production environments yet!
 
 ## Requirements
 
- - Go 1.13+
- - Postgres 9.5+
- - For Kafka (optional if using the monolith server):
-   - Unix-based system (https://kafka.apache.org/documentation/#os)
-   - JDK 1.8+ / OpenJDK 1.8+
-   - Apache Kafka 0.10.2+ (see [scripts/install-local-kafka.sh](scripts/install-local-kafka.sh) for up-to-date version numbers)
+* Go 1.13+
+* Postgres 9.5+ (if using Postgres databases)
+* Apache Kafka 0.10.2+ (optional if using the monolith server):
```

Comment:
I would rather give minimum requirements and then a separate section for polylith deployments. Mainly because I know that people will skim this, see Postgres/Kafka under "Requirements" then whine about it. If instead it said:
```
Requirements
 - Go 1.13
 - sqlite3 (or Postgres 9.5+)

If you want to run in polylith mode, you also need Kafka (0.10.2+).
```

I would omit Kafka's requirements, as they will change and this will get outdated.